examples/Makefile

```diff
@@ -1,25 +1,95 @@
+				gatk/GATK_Germline/data/ref/refSDF/seqdata0 \
+				gatk/GATK_Germline/data/ref/refSDF/seqpointer0 \
+				gatk/GATK_Germline/data/ref/refSDF/sequenceIndex0 \
+				gatk/GATK_Germline/data/ref/refSDF/summary.txt \
+				gatk/GATK_Germline/data/bams/father.bai \
+				gatk/GATK_Germline/data/bams/father.bam \
+				gatk/GATK_Germline/data/bams/mother.bai \
+				gatk/GATK_Germline/data/bams/mother.bam \
+				gatk/GATK_Germline/data/bams/motherICE.bai \
+				gatk/GATK_Germline/data/bams/motherICE.bam \
+				gatk/GATK_Germline/data/bams/motherNEX.bai \
+				gatk/GATK_Germline/data/bams/motherNEX.bam \
+				gatk/GATK_Germline/data/bams/motherRnaseq.bai \
+				gatk/GATK_Germline/data/bams/motherRnaseq.bam \
+				gatk/GATK_Germline/data/bams/motherRnaseqPP.bai \
+				gatk/GATK_Germline/data/bams/motherRnaseqPP.bam \
+				gatk/GATK_Germline/data/bams/son.bai \
+				gatk/GATK_Germline/data/bams/son.bam
+
 opencv:
 	pachctl create repo images
 	pachctl create pipeline -f opencv/edges.json
 	pachctl create pipeline -f opencv/montage.json
 	pachctl put file images@master -i opencv/images.txt
 	pachctl put file images@master -i opencv/images2.txt
 
-object_detection:
+ml/object-detection/frozen_inference_graph.pb:
+	wget -p http://download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_11_06_2017.tar.gz --directory-prefix /tmp
+	tar -C ml/object-detection --strip-components 1 -xvf /tmp/download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_11_06_2017.tar.gz ssd_mobilenet_v1_coco_11_06_2017/frozen_inference_graph.pb
+
+object_detection: ml/object-detection/frozen_inference_graph.pb
 	pachctl create repo images
 	pachctl create repo training
 	pachctl create pipeline -f ml/object-detection/model.json
 	pachctl create pipeline -f ml/object-detection/detect.json
-	pachctl put file training@master:frozen_inference_graph.pb -f ~/ssd_mobilenet_v1_coco_11_06_2017/frozen_inference_graph.pb
+	pachctl put file training@master:frozen_inference_graph.pb -f ml/object-detection/frozen_inference_graph.pb
 	pachctl put file images@master:dogs.jpg -f ml/object-detection/images/dogs.jpg
 	pachctl put file images@master:kites.jpg -f ml/object-detection/images/kites.jpg
 
+hyperparameter:
+	pachctl create repo raw_data
+	pachctl create repo parameters
+	pachctl put file raw_data@master:iris.csv -f ml/hyperparameter/data/noisy_iris.csv
+	pachctl put file parameters@master:c_parameters.txt -f ml/hyperparameter/data/parameters/c_parameters.txt --split line --target-file-datums 1
+	pachctl put file parameters@master:gamma_parameters.txt -f ml/hyperparameter/data/parameters/gamma_parameters.txt --split line --target-file-datums 1
+	pachctl create pipeline -f ml/hyperparameter/split.json
+	pachctl create pipeline -f ml/hyperparameter/model.json
+	pachctl create pipeline -f ml/hyperparameter/test.json
+	pachctl create pipeline -f ml/hyperparameter/select.json
+
+$(GATK_GERMLINE_FILES): 
+	mkdir -p gatk/GATK_Germline
```

Comment:
will this fail in an unfriendly way if we've already downloaded these files?

Reply:
I noticed the `clean` target below. Is there no way to make this friendlier so it doesn't barf if the files already exist?

Reply:
If the files are already downloaded and copied in the places I specify, the dependency targets should not trigger ($(GATK_GERMLINE_FILES) for gatk_demo and  ml/object-detection/frozen_inference_graph.pb for object_detection).  That's makefile magic!